### PR TITLE
Blog post: How container queries could help email

### PR DESCRIPTION
**Pull request by husseinalhammad**
This has been reviewed on Coda. Things that may need to be changed:

- alt text for images
- URL slug (same as markdown filename)

### CODE REVIEW

File: src/content/blog/2023/container-vs-media-queries.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+layout: "@layouts/BlogLayout"
+---
+
+Using `@media` queries in email has been commonplace for the last decade or so. Although they aren’t supported in every email client, where they do work they can really help enhance the experience for users viewing the email in a variety of viewport widths among other things. Even when they are supported they don’t always preform exactly how we need them to.
+
+Email clients embed our emails in one of 2 ways. Either the email message will be embedded in an iframe or embedded directly into the flow of the page.
+
+When embedded in an iframe, a media query in the iframe queries the width of the iframe, not the full window. So if we set `@media screen and (max-width:600px)` the styles within this media query will be applied when the iframe width is 600px or less.
+
+![Illustration of an email message within an email client UI in a browser window](/img/blog/email-client-ui-striped.png)
```
Comment (M-J-Robbins) [inline src/content/blog/2023/container-vs-media-queries.md:16]: "An email message within an email client UI. The email message container is highlighted to show the presence of an iframe, giving a viewport size around the message."

**Review by M-J-Robbins — COMMENTED** [review]
I've added some suggested alt text.

File: src/content/blog/2023/container-vs-media-queries.md (hunk 1 above)
Comment (husseinalhammad) [reply src/content/blog/2023/container-vs-media-queries.md:16]: What about the second image? Would you like the alt text for it to be updated to something else?

Comment (M-J-Robbins) [reply src/content/blog/2023/container-vs-media-queries.md:16]: Oh strange, i did add another comment too but maybe I didn't click submit. This is what I had for the other one

"An email message within an email client UI. The email message is not highlighted to show the that viewport size is the full size of the email client ui.”

Comment (husseinalhammad) [reply src/content/blog/2023/container-vs-media-queries.md]: Updated

**Pull request merged**